src/tests/functional/plugin/shared/src/behavior/ov_plugin/life_time.cpp

```diff
@@ -87,7 +87,7 @@ TEST_P(OVHoldersTest, LoadedAny) {
     {
         ov::Core core = createCoreWithTemplate();
         auto compiled_model = core.compile_model(function, targetDevice);
-        any = compiled_model.get_property(METRIC_KEY(SUPPORTED_METRICS));
+        any = compiled_model.get_property(ov::supported_properties.name());
```

Comment:
why `name()`? not a property itself?

Reply:
Here we test the life time of `any` object in users code